allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_transforms.py

```diff
@@ -227,6 +227,13 @@ def get_stimulus_templates(self) -> Optional[StimulusTemplate]:
         pkl = self._behavior_stimulus_file()
         return get_stimulus_templates(pkl=pkl)
 
+    def get_monitor_delay(self) -> float:
+        """
+        Return monitor delay for behavior only sessions
+        (in seconds)
```

Comment:
Maybe add a comment about why this is always zero as I can see folks in the future wondering why...